python/sbp/RELEASE-VERSION

```diff
@@ -1 +1 @@
-5.0.3
\ No newline at end of file
+5.0.4-alpha
```

Comment:
I'm not sure you should just bump this version, probabl yneed to do a full new release